radio/src/targets/taranis/board.cpp

```diff
@@ -47,6 +47,10 @@ extern "C" {
 }
 #endif
 
+extern void flysky_hall_stick_check_init(void);
+extern void flysky_hall_stick_init(void);
+extern void flysky_hall_stick_loop( void );
+
```

Comment:
See also: https://github.com/EdgeTX/edgetx/blob/6f4abef36a9dbd4bafca44396b08e01868cb617d/radio/src/targets/horus/board.cpp#L114-L116 and https://github.com/EdgeTX/edgetx/blob/6f4abef36a9dbd4bafca44396b08e01868cb617d/radio/src/targets/horus/board.cpp#L134-L136